### ISSUE TITLE
OPropertiesI: Domain for hasRoleRequirement

### ISSUE THREAD

**Issue by olga-kovalenko** [i40-Tools/vocabularies#3]
Currently the property hasRoleRequirement has Domain = "InternalElement". I am not 100 % sure, but I think it should also have "SUC" as a possible domain option.

**Comment by igrangel** [comment]
I'm also not 100% sure, this is taken from the [White Paper AutomationML in a Nutshell](https://www.automationml.org/o.red/uploads/dateien/1447420977-AutomationML%20in%20a%20Nutshell_151104.pdf), page 20: "The *InternalElement* shall reference at least one (but possible more than one) role class from an arbitrary role class library. Therefore, the *RoleRequirements* and the *SupportedRoleClass* sub-objects shall be applied."

**Comment by olga-kovalenko** [comment]
Yes, it means the IEs can/should refer one or more RCs, with this I 100 % agree.
But I think SUCs also can have a reference to a RC(s) via hasRoleRequirements / hasSupportedRoleClass. This sentence from the paper doesn't say that only IEs have this...

**Comment by igrangel** [comment]
This is a code from one of the examples. `<RoleRequirements RefBaseRoleClassPath="../Resource" />` and it is associated to an IE. In this case we would have: 
`aml:InternalElement1 aml:hasRoleRequirement aml:RoleRequirement1`
My question is, should we create a class `aml:RoleRequirement `or a **RoleRequirement** is always a **RoleClass (RC)**?

**Comment by olga-kovalenko** [comment]
I think the RoleRequirement is always a role class. Basically, i would say we don't need an additional class, because it adds no semantics. But this is more a personal opinion based on examples/applications I saw so far, not really a solid specification based opinion :)
